### PR TITLE
BSTOR-48 - Add actual items to the cart

### PR DESCRIPTION
**Pull request by Pacane**
Fix size calculation for the cart

### CODE REVIEW

File: client/src/main/java/com/arcbees/beeshop/client/application/product/ProductView.java [hunk 1]
```diff
@@ -159,7 +163,8 @@ public void f() {
         $(addToCart).click(new Function() {
             @Override
             public void f() {
-                getUiHandlers().onAddToCartButtonClicked();
+                int itemQuantity = Integer.parseInt(quantity.getValue());
```
Comment (olafleur) [inline client/src/main/java/com/arcbees/beeshop/client/application/product/ProductView.java:165]: Look like this is prone to error if the input `quantity` is not validated in any way?

**Comment by olafleur** [comment]
LGTM

File: client/src/main/java/com/arcbees/beeshop/client/application/product/ProductView.java (hunk 1 above)
Comment (Pacane) [inline client/src/main/java/com/arcbees/beeshop/client/application/product/ProductView.java:165]: Yeah you're right. We should probably use an IntegerBox or something like that anyway...

Comment (meriouma) [inline client/src/main/java/com/arcbees/beeshop/client/application/product/ProductView.java:165]: You can simply change to `input type="number"`

**Comment by meriouma** [comment]
LGTM

**Comment by jclariviere** [comment]
LGTM

File: client/src/main/java/com/arcbees/beeshop/client/application/product/ProductView.java (hunk 1 above)
Comment (Pacane) [inline client/src/main/java/com/arcbees/beeshop/client/application/product/ProductView.java:165]: Still, number should be validated (e.g. no negative numbers should be allowed)

Comment (meriouma) [inline client/src/main/java/com/arcbees/beeshop/client/application/product/ProductView.java:165]: `input type="number" min="1"` should work (or 0 if you can set 0 to remove)

**Pull request merged**